odoo/tools/config.py

```diff
@@ -23,6 +23,8 @@
 
 _dangerous_logger = logging.getLogger(__name__)  # use config._log() instead
 
+optparse._ = str  # disable gettext
+
```

Comment:
Side optimization, both `config.py` and `docutils` rely heavily on `optparse`. A speedscope revealed that it spent too much time trying to translate stuff. Disabled.

Reply:
:grin: